src/MidiPlayer.js

```diff
@@ -71,6 +71,7 @@ not.
             this.patchUrl = patchUrl;
             this.volume = volume;
             this.startTime = 0;
+            this.totalTime = 0; //Total Play Time 2021 03 Normal University
```

Comment:
This comment does not seem necessary. The name of the variable should be self-explanatory. You can change the name of the variable if you think it is not clear what it is for.